gateway-ha/README.md

```diff
@@ -27,3 +65,233 @@ Step 3: Add below program argument to class `HaGatewayLauncher` and debug in IDE
 ```$xslt
 server /path/to/gateway-ha/src/test/resources/config-template.yml
 ``` 
+
+## Gateway HA API
+
+### Get all backends behind the gateway
+
+`curl -X GET prestogateway.lyft.com/gateway/backend/all | python -m json.tool`
+```
+[
+    {
+        "active": true,
+        "includeInRouter": true,
```

Comment:
let's remove `includeInRouter` and `localPort` options from here.

Reply:
From the post-call? The response body has those two options.